app/models/metasploit_data_models/automatic_exploitation/run.rb

```diff
@@ -0,0 +1,14 @@
+class MetasploitDataModels::AutomaticExploitation::Run < ActiveRecord::Base
+  attr_accessible :user_id, :workspace_id, :match_set_id
```

Comment:
Put `attr_accessible` in a section called Mass Assignment Security after Associations.